package/tests/integration/playwright.config.ts

```diff
@@ -1,16 +1,16 @@
-import { defineConfig, devices } from '@playwright/test';
+import { defineConfig } from '@playwright/test';
 
 export default defineConfig({
     outputDir: './.test-results',
     preserveOutput: 'failures-only',
     forbidOnly: !!process.env.CI,
-    retries: process.env.CI ? 2 : 0,
```

Comment:
This is the default, configuring it is not necessary

Reply:
I prefer to be explicit actually.